libs/ui-toolkit/src/components/checkbox/checkbox.tsx

```diff
@@ -30,7 +31,7 @@ export const Checkbox = ({
   );
 
   return (
-    <div className="flex gap-1">
+    <label className={`flex gap-1 ${styles['label']}`} htmlFor={name}>
```

Comment:
Use the tailwind :after class

https://tailwindcss.com/docs/hover-focus-and-other-states#pseudo-elements